internal/kafka/command_quota_create.go

```diff
@@ -13,23 +13,22 @@ import (
 
 func (c *quotaCommand) newCreateCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create",
+		Use:   "create <name>",
 		Short: "Create a Kafka client quota.",
-		Args:  cobra.NoArgs,
+		Args:  cobra.ExactArgs(1),
 		RunE:  c.create,
 		Example: examples.BuildExampleString(
 			examples.Example{
 				Text: `Create client quotas for service accounts "sa-1234" and "sa-5678" on cluster "lkc-1234".`,
-				Code: `confluent kafka quota create --name clientQuota --ingress 500 --egress 100 --principals sa-1234,sa-5678 --cluster lkc-1234`,
+				Code: `confluent kafka quota create clientQuota --ingress 500 --egress 100 --principals sa-1234,sa-5678 --cluster lkc-1234`,
 			},
 			examples.Example{
 				Text: `Create a default client quota for all principals without an explicit quota assignment.`,
-				Code: `confluent kafka quota create --name defaultQuota --ingress 500 --egress 500 --principals "<default>" --cluster lkc-1234`,
+				Code: `confluent kafka quota create defaultQuota --ingress 500 --egress 500 --principals "<default>" --cluster lkc-1234`,
```

Comment:
Hmm are users actually supposed to send the string "\<default\>"??

Reply:
Yup. From the flag description: `Use "<default>" to apply the quota to all service accounts`